backend/src/server.js

```diff
@@ -30,13 +38,6 @@ class CrudServer {
     this.startListening();
   }
 
-  async startForTest() {
```

Comment:
Why you've removed it??? All auto-tests would fail now because of that!!!

Reply:
ok